File: 프로그래머스/1/12926. 시저 암호/시저 암호.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s, int n) {
    string answer = "";
    for(char c : s){
        if(isalpha(c)){
            if(isupper(c) && (c+n) > 'Z')
                answer += (c+n-26);
            else if(islower(c) && (c+n) > 'z')
                answer += (c+n-26);
            else
                answer += (c+n);
        }
        else answer += ' ';
    }
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    for c in s:
        if c.isalpha():
            for i in range(n):
                if c == 'z':
                    c = 'a'
                elif c == 'Z':
                    c = 'A'
                else:
                    c = chr(ord(c)+1)
        answer += c
    return answer